src/components/MovieCard/helper.tsx

```diff
@@ -1,29 +1,30 @@
 import * as React from 'react';
 import { ISetStateBoolean } from 'Global/types/globalTypes';
+import { IUseModalReturn } from '@/hooks/useModal/types';
 
 export const menuItems: string[] = ['Edit', 'Delete'];
```

Comment:
`['Edit', 'Delete']` would be a safer type that string[].
Learn more about tuples [here](https://www.typescriptlang.org/docs/handbook/basic-types.html#tuple)